nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -763,6 +773,27 @@ private void makeRequest(final String request, final String arguments, final Fil
         }
     }
 
+    private void sendRequest(Socket socket, Integer port, String request, String arguments, Logger logger) throws IOException {
+        logger.debug("Connecting to NiFi instance");
+        socket.setSoTimeout(60000);
+        socket.connect(new InetSocketAddress("localhost", port));
+        logger.debug("Established connection to NiFi instance.");
+        socket.setSoTimeout(60000);
+
+        logger.debug("Sending " + request + " Command to port {}", port);
```

Comment:
Could you use built-in formatting instead of concatenation?

Reply:
That's right. The fix will be in my next commit.